src/apps/service/apps/job_listings.clj

```diff
@@ -45,6 +50,7 @@
     :app_name        (:app-name job)
     :description     (:description job)
     :enddate         (job-timestamp (:end-date job))
+    :system_id       (string/lower-case (:job-type job))
```

Comment:
Perhaps for future-proofing, it'd make sense to factor it out as a variable which can be used as a function, mapping job type to system ID. For now, it can just be something like `(def job-type-to-system-id string/lower-case)`, but if later on it gets more complex it can be changed to a map or a more complex function, and that way it only needs changing one place.